rotkehlchen/externalapis/opensea.py

```diff
@@ -42,12 +45,45 @@
 log = RotkehlchenLogsAdapter(logger)
 
 
+def _read_floor_asset_price_from_collection(
+        floor_price_symbol: str,
+        collection_payment_tokens: list[dict[str, str]],
+) -> Asset:
+    """
+    Get the asset currency for the floor price reported by opensea using the list
+    of assets listed in the collection's payment tokens.
+
+    TODO: The api has a bug reported in the ticket #1455100 and the chain is reported
+    always as ethereum even if the asset is not an ethereum token.
+    When fixed adjust this function to use the chain.
+
+    May raise:
+    - UnknownAsset if the token wasn't found in the collection payment tokens.
+    """
+    for payment_token in collection_payment_tokens:
+        if payment_token['symbol'] == floor_price_symbol:
+            address = to_checksum_address(payment_token['address'])
+            with GlobalDBHandler().conn.read_ctx() as cursor:
+                asset_identifier = cursor.execute(
+                    'SELECT identifier FROM evm_tokens WHERE address=?',
```

Comment:
what if the address is more than once in the DB? Supposedly same address for different chains. It's not impossible. Current code will take the first one only.

So you consider it acceptable error until they fix on their side? If so write it here.

Reply:
Yes, I considered it acceptable until we can correctly get it from opensea

Reply:
> Yes, I considered it acceptable until we can correctly get it from opensea

So did you write it here in a comment? I told you to do so. Please read what I write. When we look at this code in the future the question will come up again.